udp_driver/src/udp_socket.cpp

```diff
@@ -23,6 +23,7 @@
 #include <vector>
 
 #include "boost/asio.hpp"
+#include <boost/system/system_error.hpp>
```

Comment:
inconsistent include

Reply:
Addressed in [86b9aae](https://github.com/autowarefoundation/transport_drivers/pull/1/commits/86b9aae8555dbb4a3103c21475d7f5e2d49981ab)